easybuild/easyconfigs/m/MEME/MEME-5.0.4-foss-2017b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.0.4'
+
+homepage = 'http://meme-suite.org'
+description = """The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or
+ GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using
+ MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate
+ motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment
+ using SpaMo or CentriMo."""
+
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['http://meme-suite.org/meme-software/%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['b5e067c8b9d9fe4a2a35d4f4d053714beb380c0c06b54ed94737dd31d93c4cf4']
+
+dependencies = [
+    ('libxml2', '2.9.8'),
```

Comment:
@murfreesboro You should stick to `libxml2` 2.9.7 to avoid introducing conflicts (same below)

Reply:
OK got it.